modules/effects/arrow-effects-data/src/test/kotlin/arrow/effects/MVarTest.kt

```diff
@@ -78,9 +78,9 @@ class MVarTest : UnitSpec() {
         binding {
           val av = mvar.empty<Int>().bind()
 
-          val f1 = av.put(10).startFiber(Dispatchers.Default).bind()
-          val f2 = av.put(20).startFiber(Dispatchers.Default).bind()
-          val f3 = av.put(30).startFiber(Dispatchers.Default).bind()
+          val f1 = av.put(10).fix().startFiber(Dispatchers.Default).bind()
```

Comment:
not sure why we need `fix()` if `startFiber` is for all `Kind<F, A> : Concurrent<F>`